doc/cephfs/cache-configuration.rst

```diff
@@ -255,10 +255,10 @@ respond to cache pressure" messages no longer appear in the logs.
 Example Scenario
 ~~~~~~~~~~~~~~~~
 
-Here is an example. A client is having 20k caps cached. At some moment the
+Here is an example. A client is having 20,000 caps cached. At some moment the
```

Comment:
Given that different locales use "," or ".", would it not be more understandable to use an SI prefix?

Reply:
What do you think about "20K"?

Reply:
The prefixes are generally case-sensitive so I think "k" is still appropriate: https://en.wikipedia.org/wiki/Metric_prefix